packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -466,7 +466,8 @@ export function applyAddItemStyle(): ICSSRules<{}> {
             background: pinkPlus,
             width: toPx(16),
             height: toPx(16),
-            left: "0"
+            left: "0",
+            top: toPx(11)
```

Comment:
fixing to 11px threw this off in chrome for me when testing. If we need to absolutely position this for all browsers, let's consider a more standard approach:

```
top: "50%",
transform: "translateY(-50%)"
```

Reply:
Wait really? I saw no change in Chrome positioning at all.